src/components/SearchBar.js

```diff
@@ -63,13 +63,13 @@ export default connect(state => ({
     console.log('on narrow screen clicked');
   }
 
-  _onSearchFocused(bool) {
-    // let searchFocused = !this.state.searchFocused;
-    // this.setState({searchFocused});
-    // this.props.onSearchFocused(searchFocused);
-    SearchAction.searchFocuse(bool);
-
-  }
+  // _onSearchFocused(bool) {
```

Comment:
This method is totally commented out, if you don't need it please delete these lines.